.gitignore

```diff
@@ -34,3 +34,4 @@
 /yarn-error.log
```

Comment:
First of all, you've not completed the task 1. This seems like you've picked parts of task 1 and 2 to solve.